projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_dag.py

```diff
@@ -4,9 +4,12 @@
 from unittest.mock import call
 from unittest.mock import MagicMock
 
+from pytest import raises
```

Comment:
Also please fix the codacy warnings. There are unused imports.

Reply:
Done.